misc/config_definitions.json

```diff
@@ -3234,10 +3246,16 @@
             "type": "array"
```

Comment:
I think you can validate these are regex with:
```
            "validate": {
                "value.*": "regex"
            }
```

Reply:
I wish there was a better rule to use...

Reply:
I guess we could define a new validation:
```
        Validator::extend('is_regex', function ($attribute, $value) {
            return @preg_match($value, null) !== false;
        });
```

Reply:
i only brought it to GUI :-)

Reply:
> I guess we could define a new validation:
> 
> ```
>         Validator::extend('is_regex', function ($attribute, $value) {
>             return @preg_match($value, null) !== false;
>         }, __(':attribute is not a valid regular expression'));
> ```

do you want me to implement this sample? If so, where?

Reply:
It goes in AppServiceProvider::bootCustomValidators()

Reply:
ah ok, thx
i'll update file and config definitions in 4-5h. I'm still at work :-)

Reply:
included :-)